cogs/resourcecog.py

```diff
@@ -0,0 +1,169 @@
+from discord.ext import commands
+import discord
+import json
+from discord.utils import get
+
+
+def jsonsave(cfg):
```

Comment:
If a function name is going to have two words in it, definitely separate them with underscores!